apps/authentication-portal/src/main/webapp/emailAddress.jsp

```diff
@@ -0,0 +1,192 @@
+<%--
+  ~ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
2021

Reply:
These are existing pages coming from the email otp endpoint. Is it fine to change them to 2021?